### ISSUE TITLE
[Bug]:  iceraven-2.25.0-browser-armeabi-v7a-forkRelease is broken

### ISSUE THREAD

**Issue by alexolog** [fork-maintainers/iceraven-browser#822]
### Steps to reproduce

Cannot install https://github.com/fork-maintainers/iceraven-browser/releases/download/iceraven-2.25.0/iceraven-2.25.0-browser-armeabi-v7a-forkRelease.apk

### Expected behaviour

APK installs

### Actual behaviour

"There was a problem while parsing the package"

### Device name

Moto G7play

### Android version

Android 10

### Iceraven version

2.25.0

**Comment by Tobi823** [comment]
Hi @alexolog 

I'm currently at vacation and therefore I cant  really debug this issue.

But it looks like that the signing certificate is somehow missing. But I dont know if this bug is caused by FFUpdater, the released APK file or the Android OS from Motorola.

https://github.com/Tobi823/ffupdater/issues/644

**Comment by Tobi823** [comment]
The APK works on my smartphone.

Just to be sure - did you use + install the arm32 version of Iceraven? https://github.com/fork-maintainers/iceraven-browser/releases/download/iceraven-2.25.0/iceraven-2.25.0-browser-armeabi-v7a-forkRelease.apk

**Comment by alexolog** [comment]
Yes, that's the one.

I also cannot update via FFUpdater, which used to work seamlessly for me for three years:

![Screenshot_20241101-125514 1](https://github.com/user-attachments/assets/7e70c2a0-ca4b-48b8-a6c9-fda73989f9a4)

**Comment by alexolog** [comment]
Yes, that's the one.

I also cannot update via FFUpdater, which used to work seamlessly for me for three years:

![Screenshot_20241101-125514 1](https://github.com/user-attachments/assets/7e70c2a0-ca4b-48b8-a6c9-fda73989f9a4)

**Comment by ascendbeing** [comment]
Can confirm: this morning I installed arm V8 just fine (other device) but armv7 doesn't install on this device. Same issue as OP. the installer is corrupted or in some way not able to be recognized as a normal valid APK file.
unfortunately there's no armeabi so I couldn't fall back to trying that; this device only takes v7a or armeabi.

**Comment by ascendbeing** [comment]
>But it looks like that the signing certificate is somehow missing. But I dont know if this bug is caused by FFUpdater, the released APK file or the Android OS from Motorola.
> 
> [Tobi823/ffupdater#644](https://github.com/Tobi823/ffupdater/issues/644)



if it matters, the v7a device I'm trying to install onto is Samsung android 13 w/ security patch from last year (because I foolishly updated past a vulnerable version that I could've rooted -- now I'm just playing the long game)

**Comment by starchivore** [comment]
Big fan of Iceraven here since I've been using it for a long time, it's the first time that error message "There was a problem while parsing the package" was displayed on my LG G4 Beat.

SHA-256 hash value of `iceraven-2.25.0-browser-armeabi-v7a-forkRelease.apk`:

```
5dd70fc8e320db978c2cdd40c0a72ebd950e89b74239671f4d1ecd4b76e65400  iceraven-2.25.0-browser-armeabi-v7a-forkRelease.apk
```
Switching to version `132.0.0` of Mull for now since it's working fine so far, that's more like a temporary deal since I still prefer Iceraven as long as a fixed release is available.

**Comment by mhbManchester** [comment]
Likewise I tried several times with iceraven-2.25.0-browser-armeabi-v7a-forkRelease.apk:
And get the parsing error message.
Look forward to a fix.

**Comment by akliuxingyuan** [comment]
should be fixed in commit 4345a4ca779b8ead112afc0e93196a77d2d45fb5, please try install CI build https://github.com/fork-maintainers/iceraven-browser/actions/runs/11611132986/artifacts/2127924731, it will be released with 132.0.1

**Comment by mhbManchester** [comment]
Yes that installed without problems.
Thanks

**Comment by akliuxingyuan** [comment]
Released with iceraven-2.15.2
